doc/radosgw/pubsub-module.rst

```diff
@@ -474,7 +475,8 @@ the events will have an S3-compatible record format (JSON):
                    "eTag":"",
                    "versionId":"",
                    "sequencer":"",
-                   "metadata":""
+                   "metadata":"",
```

Comment:
We should claim that Metadata or tags are not supported for deletion notifications and I think we should check that any new XML for notifications delete events, should not include Metadata or tags.

Reply:
agree, will ad a check to reject notifications with this combinations.
this applied to tags and metadata in the same way.
will update documentation around metadata as well.

Reply:
@ofriedma both metadata and tags are now supported in deletion notifications